src/Sylius/Component/Addressing/Matcher/ZoneMatcher.php

```diff
@@ -46,11 +37,9 @@ class ZoneMatcher implements ZoneMatcherInterface
     );
 
     /**
-     * Constructor.
-     *
-     * @param ObjectRepository $repository
+     * @param RepositoryInterface $repository
      */
-    public function __construct(ObjectRepository $repository)
+    public function __construct(RepositoryInterface $repository)
```

Comment:
To be consistent. Check all other parts of the components that use repositories.